src/NUnitConsole/nunit3-console/Options.cs

```diff
@@ -2,9 +2,14 @@
 // Options.cs
```

Comment:
I did a wholesale update of Mono Options, to the latest version from [here](https://github.com/mono/mono/blob/master/mcs/class/Mono.Options/Mono.Options/Options.cs), for a version that compiles under both .NET 2.0 and .NET Core 2.1. This is potentially a little risky, in case mono have made any breaking changes. An alternative would be to keep the existing .NET 2.0 version of the file, and use different options.cs files per build - which should avoid any breaking changes, but could potentially lead to different command line usage options between the two builds of the console. 🤔 

What do people think?

Reply:
I think it is fine to update it (hopefully we/they have tests), but it should probably be stated in the release notes, so people are aware of this.